lib/main.js

```diff
@@ -13,11 +13,12 @@ function _createWindow (options) {
     show: false
   }, options)
 
-  opts.preload = path.join(__dirname, 'renderer-preload')
   if (v[0] === 0 && v[1] >= 37) {
     opts.webPreferences = assign({
       preload: path.join(__dirname, 'renderer-preload')
     }, options.webPreferences)
+  } else {
```

Comment:
When Electron gets to version 1, this `else` condition will be (erroneously) run.